core/wiki/macros/CSS.tid

```diff
@@ -74,3 +74,47 @@ column-count: $columns$;
 \define if-background-attachment(text)
 <$reveal state="$:/themes/tiddlywiki/vanilla/settings/backgroundimage" type="nomatch" text="">$text$</$reveal>
 \end
+
+\define if-sticky-titles-calc-zindex()
+``
+[data-tiddler-title="``$(cssEscapedTitle)$``"] .tc-tiddler-title {
+	z-index: calc(500 + ``$(storyCount)$`` - ``$(tiddlerIndex)$``);
+}
+``
+\end
+
+\define if-sticky-titles()
```

Comment:
Perhaps this should just be "sticky-titles"?

Reply:
this is also changed now